js/AssessmentSet.js

```diff
@@ -397,7 +400,7 @@ export default class AssessmentSet extends ScoringSet {
   get isComplete() {
     if (this.isAwaitingChildren || !this.isAvailable) return false;
     if (this.attempt?.isInSession) return this.isAttemptComplete;
-    if (this.hasSoftReset) return this.attempts.wasComplete;
+    if (this.iSoftReset) return this.attempts.wasComplete;
```

Comment:
```suggestion
    if (this.isSoftReset) return this.attempts.wasComplete;
```

`isSoftReset` not `iSoftReset` throughout

Reply:
Thanks for spotting. Had is typed properly in https://github.com/adaptlearning/adapt-contrib-scoringAssessment/pull/6 which was closed as ended up as a duplicate of this PR, so I shouldn't have made this mistake.